src/semeio/forward_models/__init__.py

```diff
@@ -0,0 +1,160 @@
+# class notation
+from ert.config.forward_model_step import ForwardModelStepJSON, ForwardModelStep
+
+
+class Design2Params(ForwardModelStep):
+    def __init__(self):
+        super().__init__(
+            name="DESIGN2PARAMS",
+            executable="design2params",
```

Comment:
Potentially, in order to mimic `subprocess.run` it would be convenient to specify the cmd_line args:

```python
cmd=["design2params", "<IENS>", "<xls_filename>", "<designsheet>", "<defaultssheet>"],
```

Reply:
I believe subprocess allows both list and str.

Reply:
Did this, stuck to only list syntax instead of str, even though both are possible